src/T2DMap.cpp

```diff
@@ -2864,54 +2860,51 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
                     connect(unlockRoom, &QAction::triggered, this, &T2DMap::slot_unlockRoom);
                     popup->addAction(unlockRoom);
                 }
 
                 if (selectionSize > 0) {
-                    auto weightRoom = new QAction(tr("Weight", "2D Mapper context menu (room) item"), this);
-                    weightRoom->setToolTip(tr("Set room weight", "2D Mapper context menu (room) item tooltip"));
+                    auto weightRoom = new QAction(tr("Set room weight...", "2D Mapper context menu (room) item", selectionSize), this);
                     connect(weightRoom, &QAction::triggered, this, &T2DMap::slot_setRoomWeight);
                     popup->addAction(weightRoom);
                 }
 
                 if (selectionSize > 0) {
-                    auto deleteRoom = new QAction(tr("Delete", "2D Mapper context menu (room) item"), this);
-                    deleteRoom->setToolTip(tr("Delete room", "2D Mapper context menu (room) item tooltip"));
+                    auto deleteRoom = new QAction(tr("Delete room(s)", "2D Mapper context menu (room) item", selectionSize), this);
                     connect(deleteRoom, &QAction::triggered, this, &T2DMap::slot_deleteRoom);
                     popup->addAction(deleteRoom);
                 }
 
                 if (selectionSize > 0) {
-                    auto moveRoomXY = new QAction(tr("Move to", "2D Mapper context menu (room) item"), this);
-                    moveRoomXY->setToolTip(tr("Move selected group to a given position", "2D Mapper context menu (room) item tooltip"));
+                    auto moveRoomXY = new QAction(tr("Move room(s) to position...", "2D Mapper context menu (room) item", selectionSize), this);
+                    moveRoomXY->setToolTip(tr("Move selected room or group of rooms to a given position", "2D Mapper context menu (room) item tooltip"));
                     connect(moveRoomXY, &QAction::triggered, this, &T2DMap::slot_movePosition);
                     popup->addAction(moveRoomXY);
                 }
 
                 if (selectionSize > 0) {
-                    auto roomArea = new QAction(tr("Area", "2D Mapper context menu (room) item"), this);
-                    roomArea->setToolTip(tr("Set room's area number", "2D Mapper context menu (room) item tooltip"));
+                    auto roomArea = new QAction(tr("Move room(s) to area...", "2D Mapper context menu (room) item", selectionSize), this);
                     connect(roomArea, &QAction::triggered, this, &T2DMap::slot_setArea);
                     popup->addAction(roomArea);
                 }
 
-                auto createLabel = new QAction(tr("Create Label", "2D Mapper context menu (room) item"), this);
+                auto createLabel = new QAction(tr("Create new label...", "2D Mapper context menu (room) item"), this);
                 createLabel->setToolTip(tr("Create labels to show text or images", "2D Mapper context menu (room) item tooltip"));
```

Comment:
Actually the plurals here should not be - this only does one label per invocation IIRC.
```suggestion
                createLabel->setToolTip(tr("Create label to show text or an image", "2D Mapper context menu (room) item tooltip"));
```

Reply:
Funny you comment on the tool-tip that I did not even touch this time around 😄